integration/spark/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -332,12 +332,26 @@ object AlterTableUtil {
       tableProperties: mutable.Map[String, String],
       oldColumnName: String,
       newColumnName: String): Unit = {
+    val columnProperties = Seq("NO_INVERTED_INDEX",
+      "INVERTED_INDEX",
+      "INDEX_COLUMNS",
+      "COLUMN_META_CACHE",
+      "DICTIONARY_INCLUDE",
```

Comment:
we have removed  DICTIONARY_INCLUDE, DICTIONARY_EXCLUDE from 2.0, please check the applicable table properties and keep only those also add if something new added in 2.0

Reply:
Also INDEX_COLUMNS are changed to SPATIAL_INDEX I think, refer docs/spatial-index-guide.md

Reply:
Removed "DICTIONARY_INCLUDE" and "DICTIONARY_EXCLUDE". All properties which related to column name are now included in the list. 
"INDEX_COLUMNS" and "SPATIAL_INDEX" is the property only for bloom index and spatial index table, which these two kinds of table are blocked to rename the column, so it is ok for these two properties not to be included in the property list.